Subject: A missing DESTDIR
Author: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Bug: http://bugs.digium.com/view.php?id=14701
Last-Update: 2009-12-19

Accepted in upstream rev 1077 and will be included in next asterisk-addons
1.6.2.x release.

--- a/Makefile
+++ b/Makefile
@@ -185,7 +185,8 @@ install: _all $(SUBDIRS_INSTALL) install
 
 install-xmldoc: doc/addons-en_US.xml
 	@echo "Installing XML documentation"
-	@$(INSTALL) -m 644 doc/addons-*.xml $(ASTDATADIR)/documentation
+	install -d $(DESTDIR)$(ASTDATADIR)/documentation
+	@$(INSTALL) -m 644 doc/addons-*.xml $(DESTDIR)$(ASTDATADIR)/documentation
 
 cleantest:
 
